### PR TITLE
#162948878 Post Comment

### PR DESCRIPTION
**Pull request by Deekerubo**
#### What does this PR do?
This adds the functionality necessary to post comments on articles.

#### Description of Task to be completed?
* [x]  Build the UI elements necessary for `Reading comments`
* [x]  Add the UI elements necessary for `Creating comments`
* [x]  Write out the functionality for the above

#### How should this be manually tested?
1. Run the following commands in Terminal
2. Execute the following commands in Terminal:

```
git clone https://github.com/andela/ah-alpha-frontend.git
git checkout ft-comment-post-162948878
cd ah-alpha-frontend

```
1. Continue in `Terminal`

```

npm install
npm start
```
1. The app auto-opens on your browser at `localhost:3000`
2. `Login` or `Register` and you will be redirected to the articles page.
3. Click on an article.  Beneath each article, there is a comments section, where a user can `Create` comments on articles.
screenshot 2019-02-22 at 10 52 06

### CODE REVIEW

File: src/actions/getOneArticleAction.js [hunk 1]
```diff
@@ -1,21 +1,24 @@
 import axios from "axios";
-import { BASE_URL } from "../actions/constants";
-import * as profileActions from "./profileActions";
+import * as profileActions from "./profileActions"
```
[the remark this answers is not in the document]
Comment (danuluma) [reply src/actions/getOneArticleAction.js]: Please fix this...add a semi-colon

**Comment by jamesbeamie** [comment]
- Kindly resolve this merge conflicts

File: src/components/comment/addComment.js [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+// Need to be globally available
+let elem = "";
+let errorSubscript = "";
+// eslint-disable-next-line no-var
+
+class AddCommentComponent extends Component {
+  state = {
+    comment: ""
+  }
+
```
[the remark this answers is not in the document]
Comment (jamesbeamie) [reply src/components/comment/addComment.js:23]: Just have a single blank line

File: src/components/common/Navbar.js [hunk 3]
```diff
@@ -52,4 +52,4 @@ class Navbar extends React.Component {
   }
 }
 
-export default Navbar;
+export default Navbar;
```
[the remark this answers is not in the document]
Comment (danuluma) [reply src/components/common/Navbar.js]: Add a new line at the end of the file

**Review by danuluma — APPROVED** [review]

**Review by jamesbeamie — APPROVED** [review]

**Review by boltonotieno — APPROVED** [review]
LGTM

**Pull request merged**